test/partitioning.jl

```diff
@@ -82,9 +82,39 @@
     @test length(p) == 4
     @test all(nelements.(p) .== 25)
 
+    p = partition(grid, BlockPartition(T(5),T(5), neighbors = false))
+    @test length(p) == 4
+    @test all(nelements.(p) .== 25)
```

Comment:
This test is redundant right? It already happens right above. Please remove it.

Reply:
Removed!